docset/winserver2025-ps/smbshare/Get-SmbClientAccessToServer.md

```diff
@@ -0,0 +1,195 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbClientAccessToServer.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 02/22/2024
+online version: https://learn.microsoft.com/powershell/module/smbshare/get-smbclientaccesstoserver?view=windowsserver2025-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-SmbClientAccessToServer
+---
+
+# Get-SmbClientAccessToServer
+
+## SYNOPSIS
+Gets information about SMB client access to a specified server.
+
+## SYNTAX
+
+```
+Get-SmbClientAccessToServer [-Name] <String[]> [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>]
+ [-AsJob] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+```
+Get-SmbClientAccessToServer -InputObject <CimInstance[]> [-CimSession <CimSession[]>]
+ [-ThrottleLimit <Int32>] [-AsJob] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-SmbClientAccessToServer` cmdlet is used to retrieve information about SMB client access to
+a specified server. When you run this cmdlet, it'll return a list of all SMB client connections to
+the specified server.
+
+## EXAMPLES
+
+### Example 1: Retrieve client access information
+
+```powershell
+Get-SmbClientAccessToServer -Name "Server01"
+```
+
+This command retrieves the SMB client access information for a device named "Server01".
+
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](/powershell/module/cimcmdlets/new-cimsession) or
+[Get-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+
+Prompts you for confirmation before running the cmdlet.
+
```

Comment:
**Confirm** and **WhatIf** parameters go at the end of the list before CommonParameters.